SPECS/kernel-alt.spec

```diff
@@ -408,7 +408,15 @@ Patch344: 0004-xen-xenbus-xen_bus_type-Support-will_handle-watch-ca.patch
 Patch345: 0005-xen-xenbus-Count-pending-messages-for-each-watch.patch
 Patch346: 0006-xenbus-xenbus_backend-Disallow-pending-watch-message.patch
 Patch347: xsa350-linux.patch
-
+Patch348: xsa361-linux-1.patch
+Patch349: xsa361-linux-2.patch
+Patch350: xsa361-linux-3.patch
+Patch351: xsa361-linux-4.patch
+#Patch352: xsa362-linux-1.patch
```

Comment:
Why are two patches disabled?

Reply:
There was a conflict and error on my side in resolving it. Now they are back and conflicting patch has been modified.